extensions/positron-python/python_files/posit/positron/positron_jedilsp.py

```diff
@@ -844,63 +937,32 @@ def positron_did_close_diagnostics(
     return did_close_diagnostics(server, params)
 
 
-@debounce(1, keyed_by="uri")
```

Comment:
Reworked this a little to patch their function so we could copy paste less elsewhere